src/dataflow-types/src/client.rs

```diff
@@ -191,6 +195,10 @@ pub enum StorageCommand<T = mz_repr::Timestamp> {
         /// A list of updates to be introduced to the input.
         updates: Vec<Update<T>>,
     },
+    /// Truncate the contents of a table
+    /// TODO: This command only exists because coord currently wants to initialize several tables
+    /// at startup.
+    Truncate { id: GlobalId },
```

Comment:
Not a big deal, but I wonder if we eventually want `UpdateAndDowngrade(TableOp)` or something analogous. Decouple the vocabulary of updates from the set of commands? Not pressing at the moment, but in terms of narrowing the cognitive overhead of the commands.

Reply:
Elsewhere, we usually have batch commands, e.g. `ids: Vec<GlobalId>`. No strong feelings, but is there a reason not to do that here (and simplify the `truncate_tables` implementation later)?